types/stripe/index.d.ts

```diff
@@ -1352,8 +1352,9 @@ declare namespace Stripe {
             /**
              * For most Stripe users, the source of every charge is a credit or debit card.
              * This hash is then the card object describing that card.
+             * This value is null if this charge is associated with a Payment Intent instead of a Source.
              */
-            source: IStripeSource;
```

Comment:
The latest documentation for Charges at https://stripe.com/docs/api/charges/object doesn't even mention source anymore, but looking at the actual API requests it's still there but set to `null` if you are using payment methods.